src/CycloneDX.Core/Xml/Serializer.Serialization.cs

```diff
@@ -24,6 +24,7 @@
 using System.Xml;
 using System.Xml.Serialization;
 using CycloneDX.Models;
+using ProtoBuf;

```

Comment:
This line is no longer needed.

Reply:
Should be fixed by https://github.com/CycloneDX/cyclonedx-dotnet-library/pull/354.